script.js

```diff
@@ -0,0 +1,115 @@
+(function () {
+  let lastId = 0;
+  const taskWrapper = document.getElementById('task_wrapper');
+  const btnSave = document.getElementById('save_task');
+  let removeIcon;
+  let taskList;
+
+  function init() {
+    if (window.localStorage.getItem('taskList')) {
+      taskList = JSON.parse(window.localStorage.getItem('taskList'));
+    } else {
+      taskList = [];
+    }
+    /* eslint-disable no-use-before-define */
```

Comment:
Kindly work on your Linter Rule provided around your project is disabled in every moment. Remove your es-lint disable and work improving your code.